cmd/build/helmify/kustomization.yaml

```diff
@@ -5,149 +5,155 @@ commonLabels:
   release: "{{ .Release.Name }}"
   heritage: "{{ .Release.Service }}"
 bases:
-  - "../../../config/default"
+- "../../../config/default"
 patchesStrategicMerge:
-  - kustomize-for-helm.yaml
-  - delete-ports.yaml
+- kustomize-for-helm.yaml
+- delete-ports.yaml
 patchesJson6902:
-  - target:
```

Comment:
Can we keep consistent indentation here so we can more easily see what the diff is?

Reply:
Agh sorry. My stupid IDE does not persist my file ignore settings to RAM to I have to re-add them whenever I restart my computer lol :(